scarb/src/bin/scarb/args.rs

```diff
@@ -166,6 +166,8 @@ pub enum Command {
     /// Manipulate packages cache.
     #[clap(subcommand)]
     Cache(CacheSubcommand),
+    /// Compile until diagnostics checking.
```

Comment:
The `diagnostics checking` sounds kind of cryptic to me. Note, that this will be shown to users on `--help`. Maybe we could use something more similar to what Cargo does?
```
    check, c    Analyze the current package and report errors, but don't build object files
```